frame/nomination-pools/rpc/src/lib.rs

```diff
@@ -0,0 +1,99 @@
+// This file is part of Substrate.
+
+// Copyright (C) 2022 Parity Technologies (UK) Ltd.
+// SPDX-License-Identifier: Apache-2.0
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// 	http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! RPC interface for the nomination-pools pallet.
+
+pub use self::gen_client::Client as NominationPoolsClient;
+use codec::Codec;
+use jsonrpc_core::Error;
+use jsonrpc_derive::rpc;
+pub use pallet_nomination_pools_rpc_runtime_api::NominationPoolsApi as NominationPoolsRuntimeApi;
+use pallet_nomination_pools_rpc_runtime_api::NpApiError;
+use sp_api::ProvideRuntimeApi;
+use sp_blockchain::HeaderBackend;
+use sp_runtime::{generic::BlockId, traits::Block as BlockT};
+use std::sync::Arc;
+
+trait ToError {
+	fn to_error(&self) -> Error;
+}
+
+impl ToError for NpApiError {
+	fn to_error(&self) -> Error {
+		match self {
+			NpApiError::MemberNotFound => Error {
+				code: jsonrpc_core::ErrorCode::ServerError(2),
+				message: "Member with the given account was not found.".to_string(),
+				data: None,
+			},
+			NpApiError::OverflowInPendingRewards => Error {
+				code: jsonrpc_core::ErrorCode::ServerError(3),
+				message: "An overflow occured when calculating the pending rewards.".to_string(),
+				data: None,
+			},
+		}
+	}
+}
+
+#[rpc]
+pub trait NominationPoolsRpc<BlockHash, AccountId, ResponseType> {
+	#[rpc(name = "nompools_pending_rewards")]
```

Comment:
I believe the better way to name this is `nominationPools_pendingRewards` -- double check with other rpc endpoints, you should be able to see what is the idiom.

Reply:
or, `npools_pendingRewards`